src/compiler/factory.ts

```diff
@@ -1252,7 +1252,16 @@ namespace ts {
         return (node.expression as StringLiteral).text === "use strict";
     }
 
+    /**
+     * Add any necessary prologue-directives into target statement-array.
+     * The function needs to be called during each transformation steps.

```

Comment:
*step
Also, you should probably modified the comment to mention that it needs to be called whenever we transform the statement list of a source file, namespace, or function-like body.